examples/nodejs/src/config.ts

```diff
@@ -19,8 +19,8 @@ const configGuard = t.type({
 
 export type Config = t.TypeOf<typeof configGuard>;
 
```

Comment:
the documentation in the README is the following : 
```
{
  // We use blockfrost for the lucid wallet, you can create a projectId here
  // https://blockfrost.io/
  "blockfrostProjectId": "YOUR_PROJECT_ID",
  "blockfrostUrl": "https://cardano-preprod.blockfrost.io/api/v0",
  "blockfrostNetwork": "Preprod",
  // You can create this seed phrase from any wallet. Do not reuse a real wallet phrase
  // for a test example.
  "seedPhrase": "alpha beta delta...",
  "runtimeURL": "<url to a runtime instance>"
}
```
and the associated guard is  : 
```
const configGuard = t.type({
  blockfrostProjectId: t.string,
  blockfrostUrl: t.string,
  network: lucidNetworkGuard,
  seedPhrase: t.string,
  runtimeURL: t.string,
});
```
the documentation is saying `blockfrostNetwork` instead of `network`, there is a mismatch